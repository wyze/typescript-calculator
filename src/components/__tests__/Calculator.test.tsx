import { h } from 'preact'
import Calculator from '../Calculator'
import snapshot from './utils/snapshot'

describe('<Calculator />', () => {
  it('renders', () => snapshot(<Calculator />))
  it('renders with children', () => snapshot(<Calculator><span /></Calculator>))
})
