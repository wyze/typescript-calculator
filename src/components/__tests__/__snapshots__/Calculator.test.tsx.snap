// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calculator /> renders 1`] = `
<div
  className="css-zzci37"
/>
`;

exports[`<Calculator /> renders with children 1`] = `
<div
  className="css-zzci37"
>
  <span />
</div>
`;
